### PR TITLE
fix: add missing copyrights for python3*venv

### LINKED ISSUE #272: missing copyright slice for python3.*-venv
https://github.com/canonical/chisel-releases/blob/ff34d1fda124dd906492a4ebe52df9da85e15d76/slices/python3.11-venv.yaml#L2

@gregory-schiano I believe we unintentionally left out the copyright slice, which in this case is a symlink, so it should look something like

```yaml
/usr/share/doc/python3.12-venv: {symlink: /usr/share/doc/python3.12/copyright} 
# or even 
# /usr/share/doc/python3.12-venv: 
# but that could bring additional file that we don't need
```

### PR DESCRIPTION
**Pull request by cjdcordeiro**
# Proposed changes
Fixes #272 

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/python3.10-venv.yaml [hunk 1]
```diff
@@ -9,3 +12,9 @@ slices:
       - python3.10_standard
     contents:
       /usr/lib/python3.10/ensurepip/*.py:
+
+  copyright:
+    essential:
+      - python3.10_copyright
+    contents:
+      /usr/share/doc/python3.10-venv/: { symlink: /usr/share/doc/python3.10/ }
```
Comment (rebornplusplus) [inline slices/python3.10-venv.yaml:20]:
```suggestion
      /usr/share/doc/python3.10-venv/:
```

This already comes as a symlink from the package.

```sh
lrwxrwxrwx root/root         0 2024-11-07 02:22 ./usr/share/doc/python3.10-venv -> python3.10
```

File: slices/python3.11-venv.yaml [hunk 2]
```diff
@@ -9,3 +12,9 @@ slices:
       - python3.11_standard
     contents:
       /usr/lib/python3.11/ensurepip/*.py:
+
+  copyright:
+    essential:
+      - python3.11_copyright
+    contents:
+      /usr/share/doc/python3.11-venv/: { symlink: /usr/share/doc/python3.11/ }
```
Comment (rebornplusplus) [inline slices/python3.11-venv.yaml:20]: Same as `python3.10-venv` above.

```suggestion
      /usr/share/doc/python3.11-venv/:
```

**Review by rebornplusplus — CHANGES_REQUESTED** [review]

**Comment by cjdcordeiro** [comment]
@rebornplusplus applied changes

**Review by rebornplusplus — APPROVED** [review]

**Review by linostar — APPROVED** [review]
LGTM.

**Pull request merged**